### PR TITLE
Visegradi Clothes PR

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
This PR adds a sweater, a nyakas, and a ponczo. Credits to Wezzy for nyakas and ponczo sprites.

_(edited by SleepyGemmy @ 2023-03-17: updated PR description.)_

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!wip

**Comment by TheDanseMacabre** [comment]
!review

**Review by FluffyGhoster — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
the nyakas will need to be made into a contained sprite.

**Comment by TheDanseMacabre** [comment]
> the nyakas will need to be made into a contained sprite.

done

File: code/modules/clothing/head/misc_special.dm [hunk 1]
```diff
@@ -135,14 +135,25 @@
...
 
+/obj/item/clothing/head/ushanka/nyakas
+	name = "visegradi nyakas"
+	desc = "A type of flap hat that is extremely popular on Visegrad. It is designed to keep one's head and neck dry, and the flap can be pinned to the sides of the hat when not needed."
+	contained_sprite = TRUE
+	icon = 'icons/obj/item/clothing/head/nyakas.dmi'
+	icon_state = "nyakas"
+	item_state = "nyakas"
+	
+
```
Comment (SleepyGemmy) [inline code/modules/clothing/head/misc_special.dm:146]:
```suggestion
```

**Pull request merged**